sdmetrics/single_table/privacy/loss.py

```diff
@@ -36,7 +38,8 @@ class InverseCDFDistance(LossFunction):
     """
 
     def __init__(self, p=2):
-        """
+        """Set up the appropriate attributes.
```

Comment:
For `__init__` methods I think it's better to just not have a docstring at all and rely on the class docstring only, otherwise `Sphinx` has trouble rendering the right thing. So, what I would do is move the `Args` section to the class `docstring`, so Sphinx can find and render it, and remove this docstring altogether.
Notice that this will be accepted by the flake8 addon, so it won't complain about the `__init__` not having a docstring either.